lib/v2/utils/debouncer.dart

```diff
@@ -0,0 +1,18 @@
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+
+class Debouncer {
```

Comment:
@gguijarro-c-chwy This case is not similar to what you did before to cancel events and only the last login is executed by the user.